packages/create-neutrino-project/commands/init/generator/questions.js

```diff
@@ -0,0 +1,140 @@
+const path = require('path');
+
+module.exports = (defaults) => [
+  {
+    name: 'project',
+    type: 'list',
+    message: 'Do you want to build an application, a library, or components?',
+    choices: [
+      { name: 'Application',
```

Comment:
Some of these properties are on the same line as the brace; move them to their own line.